.gitignore

```diff
@@ -1,9 +1,22 @@
+# Environment files
+.env
 .env-back
 .env-front
+.envrc
 
+# citizenlab-ee files
 back/engines/ee
-node_modules
```

Comment:
Not sure about `node_modules`. Should I leave it in the top-level file as well?

Reply:
`node_modules` should never be in the root folder, but what sometimes happened (including to me) is that you would do `npm install` in the root folder by accident, which would then create a `node_modules` folder there. This folder would then get added to the git repo. IIRC we decided to put it in .gitignore because of this

Reply:
Yes, to me it makes sense to leave it in.

Reply:
A comment wouldn't hurt! :)